src/App.tsx

```diff
@@ -64,6 +71,10 @@ import {Loading} from 'components/views/loading';
 
 enableScreens();
 
+YellowBox.ignoreWarnings([
+  'Warning: componentWillReceiveProps has been renamed'
+]);
```

Comment:
I had to disable this warning because the yellow box covers the Continue button in the "Your Data" screen of the Onboarding flow, blocking some tests.

Reply:
Why not move these to the UNSAFE variants for the time being?

Reply:
Thanks @dharding for the comment. I cannot, because that method is used in an imported library, not in the app code. The warning says:

```
Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://fb.me/react-unsafe-component-lifecycles for details.

...

Please update the following components: Markdown
```

The [`src/components/atoms/markdown.tsx`](https://github.com/covidgreen/covid-green-app/blob/add_e2e_tests/src/components/atoms/markdown.tsx#L46) does not use that method but the imported [`M` object from `react-native-easy-markdown` does](https://github.com/TitanInvest/react-native-easy-markdown/blob/master/index.js#L36). 

And we are already using the latest version of the library (`2.0.0`). There isn't an updated version that fixes it.